drone/Dockerfile

```diff
@@ -1,10 +1,24 @@
-FROM rust:buster as build
+FROM rust:latest as build
```

Comment:
Why do this in the drone but not controller?

Reply:
oops oversight